src/Pages/CartPage.java

```diff
@@ -0,0 +1,10 @@
+package Pages;
+
+public class CartPage {
+	
+		public static void doCart() {
+		
+		System.out.println("Cart Page");
```

Comment:
Hi Janani,

Kindly following the below changes.
Added cart page logic

Reply:
Done karthik.